### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/gtk-rs/gir/issues/845

cc @sdroege @EPashkin

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Next PRs will come in the next days. :)

File: gtk-sys/LICENSE [hunk 1]
```diff
@@ -1,22 +0,0 @@
-The MIT License (MIT)
-
-Copyright (c) 2013-2015, The Gtk-rs Project Developers.
```
Comment (EPashkin) [inline gtk-sys/LICENSE]: Why it was deleted?

File: atk-sys/Cargo.toml [hunk 2]
```diff
@@ -34,5 +34,5 @@ name = "atk-sys"
 repository = "https://github.com/gtk-rs/sys"
 version = "0.9.0"
 
-["package.metadata.docs.rs"]
-features = ["dox", "v2_32"]
+[package.metadata.docs.rs]
```
Comment (EPashkin) [inline atk-sys/Cargo.toml]: Strange, but my regen don't have empty line before this.
Any idea why?

Comment (GuillaumeGomez) [reply atk-sys/Cargo.toml]: Unfortunately no. I had to make the changes by hand. I think it's a toml bug but not sure. We should check if updating the toml version fixes this bug. Don't have time now but will definitely look into it later.

File: gtk-sys/LICENSE (hunk 1 above)
Comment (GuillaumeGomez) [reply gtk-sys/LICENSE]: Arf, a forgotten experiment!

**Comment by EPashkin** [comment]
So empty line can't be added?
Then IMHO sourceview-sys need update too.
👍

**Comment by GuillaumeGomez** [comment]
I'll check into `toml` directly but not in this PR. :)

**Comment by GuillaumeGomez** [comment]
And `sourceview-sys` will be updated in any case. :)

**Pull request merged**